scripts/periodic/deactivate_requested_accounts.py

```diff
@@ -0,0 +1,57 @@
+"""
+This script runs nightly and emails users who want to delete there account with info on how to do so. Users who don't
+have any content can be automatically deleted.
+"""
+import sys
+import logging
+
+from website import mails
+from django.utils import timezone
+
+from framework.celery_tasks import app as celery_app
+from website.app import setup_django
+setup_django()
+from osf.models import OSFUser
+from website.settings import OSF_SUPPORT_EMAIL, OSF_CONTACT_EMAIL
+
+from scripts.utils import add_file_logger
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def deactivate_requested_accounts(dry_run=True):
+    users = OSFUser.objects.filter(requested_deactivation=True, contacted_deactivation=False, date_disabled__isnull=True)
```

Comment:
Thinking through this...suggestions welcome, but I think with this logic a user can only request deactivation once.  Say I request deactivation and this script runs, which will mark `contacted_deactivation` as True.  Say I have non-deleted projects, so the help desk is emailed.  I then cancel my deactivation request, so Sara doesn't delete my user.   At any point in the future, I won't be able to request again because contacted_deactivation is True.

I'm wondering if a user cancels their deactivation request if we should also set `contacted_deactivation` back to False.